docs/changes.rst

```diff
@@ -34,12 +34,23 @@ Released: not yet
 * Removed the deprecated `schema_mof_file` property in `DMTFCIMSchema`, in favor
   of the `schema_pragma_file` property. (See issue #2284)
 
+* Changed the handling of invalid types of input parameters to WBEMConnection
```

Comment:
I think that as much as standardization, it extended the type checking so that operations that might have been accepted even with invalid parameters are now caught by.t the client code. This could cause incompatibilites with older versions that ignored certain invalid parameters.

Reply:
What are you suggesting in terms of a change to the current text in the change history?

Reply:
“The May cause TypeErrors in sending client requests if parameters are incorrectly typed that could have slipped through the client and XML generation code before these strict type tests on all parameters were added.

Reply:
Something like “this may cause type errors in requests because of the new strict type checking “

Reply:
The text already says that TypeError is now raised for invalid types. I tried to fit your sentence in, but any way I tried, it sounded pretty much like a duplication of the existing text about TypeError. I decided not to change it.